fake-vm-component-scripts/fake-vm/launch.py

```diff
@@ -10,25 +10,30 @@
 
 instances = []
 
-def fix_config_keys(source):
-    for k, v in source:
-        yield (k, v) if '.' not in k else (k.split('.')[1], v)
+config_keys = {
+    k: k.split('.')[1] for k in {
+        'configuration.name',
+        'configuration.address',
+        'configuration.login',
+        'configuration.failure',
```

Comment:
For discovery testing you have `--name`. I'm ok with having name in the python model, I don't like `configuration.name` parameter. When we add instance context to python inputs we will be able to set this name from instance name. For now let's have python names for discovered instances only.
As for other configuration parameters:
 * `configuration.address` — not needed. Address should be generated in `launch.py` with an option to modify with `manage.py`.
 * `configuration.login` — needed.
 * `configuration.failure` — not needed. Failures should be introduced by `manage.py` only.